### PR TITLE
Add CRUD privileges management

### PR DESCRIPTION
**Pull request by DiogoTx**
Add documentation to explain the privileges required to execute crud operations

### CODE REVIEW

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+keywords: omnia3
+summary: "Privileges"
+sidebar: omnia3_sidebar
+permalink: omnia3_privilegesmanagement.html
+folder: omnia3
+---
+
+## 1. Introduction
+
+On OMNIA Platform, Privileges control the accessbility to the CRUD (Create, Read, Update and Delete) operations of a given entity.
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md:12]: Within OMNIA, Privileges...

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+### 1.1 Read
+
+The following privileges combination is recommended to list an entity:
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md:17]: The following combination of privileges...

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+- [EntityCode]Dashboard_Read:  This privilege grants access to view the default entity dashboard
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md:19]: This privilege grants viewing access to an entity's default dashboard

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+- [EntityCode]Query_Read: This privilege grants access to execute the default entity query. Access to other entity queries must also be granted is needed
+- [EntityCode]_Read: This privilege grants access to read the entity data
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md:21]: Grants reading access to the entity's data

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+
+### 1.2 Create/Update
+
+To create or update an entity, Read privileges are required, along with:
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md:26]: remove ","

Comment (daniel8lopes) [reply docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md:26]: "alongside with"

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+- [EntityCode]_Write: This privilege grants access to create and update entities.
+
+
+### 1.3 Delete
+
+To delete an entity, Read privileges are required, along with:
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md:33]: "alongside with"

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+- [EntityCode]_Delete: This privilege grants access to read the entity data
+
+
+### 1.4 Destroy Sensitive Data
+
+To destroy sensitive data of an entity, Read privileges are required, along with:
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md:40]: alongside with

**Review by daniel8lopes — COMMENTED** [review]
single comments added

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md (hunk 1 above)
Comment (DiogoTx) [reply docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md]: Reviewed

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md (hunk 3 above)
Comment (DiogoTx) [reply docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md]: Reviewed

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md (hunk 5 above)
Comment (DiogoTx) [reply docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md]: Reviewed

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md (hunk 7 above)
Comment (DiogoTx) [reply docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md]: Reviewed

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md (hunk 6 above)
Comment (DiogoTx) [reply docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md]: Reviewed

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md (hunk 4 above)
Comment (DiogoTx) [reply docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md]: Reviewed

File: docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md (hunk 2 above)
Comment (DiogoTx) [reply docs/pages/omnia3/Authorization/omnia3_privilegesmanagement.md]: Reviewed

**Pull request merged**